packaging/rpm/scripts/pre-install.sh

```diff
@@ -6,11 +7,11 @@ set -e
 #   * on upgrade: $1 > 1
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax
 
-PACKS_GROUP=%{packs_group}
-SYS_USER=%{stanley_user}
-ST2_USER=%{svc_user}
```

Comment:
These macros were defined in `st2.spec`: https://github.com/StackStorm/st2-packages/blob/9974d04084b01cde9d11994f02abf46290c37fe3/packages/st2/rpm/st2.spec#L3-L5

The equivalent hard-coded values were already embedded in the deb maintainer scripts, so, this does not change anything. It only inlines the values for rpm.